pom.xml

```diff
@@ -45,22 +45,25 @@
     <failsafe.maven.plugin.version>2.22.0</failsafe.maven.plugin.version>
     <findbugs.maven.plugin.version>3.0.5</findbugs.maven.plugin.version>
     <hotels.oss.plugin.config.version>1.2.2</hotels.oss.plugin.config.version>
-    <jacoco.version>0.8.2</jacoco.version>
+    <jacoco.version>0.8.4</jacoco.version>
     <jdk.version>1.8</jdk.version>
     <license.maven.plugin.version>3.0</license.maven.plugin.version>
-    <maven.checkstyle.plugin.version>3.0.0</maven.checkstyle.plugin.version>
-    <maven.compiler.plugin.version>3.8.0</maven.compiler.plugin.version>
+    <maven.checkstyle.plugin.version>3.1.0</maven.checkstyle.plugin.version>
+    <maven.compiler.plugin.version>3.8.1</maven.compiler.plugin.version>
     <maven.dependency.plugin.version>3.1.1</maven.dependency.plugin.version>
-    <maven.jar.plugin.version>3.1.0</maven.jar.plugin.version>
-    <maven.javadoc.plugin.version>3.0.1</maven.javadoc.plugin.version>
-    <maven.jxr.plugin.version>2.5</maven.jxr.plugin.version>
-    <maven.pmd.plugin.version>3.10.0</maven.pmd.plugin.version>
+    <maven.jar.plugin.version>3.1.2</maven.jar.plugin.version>
+    <maven.javadoc.plugin.version>3.1.0</maven.javadoc.plugin.version>
+    <maven.jxr.plugin.version>3.0.0</maven.jxr.plugin.version>
+    <maven.pmd.plugin.version>3.12.0</maven.pmd.plugin.version>
     <maven.project.info.reports.plugin.version>3.0.0</maven.project.info.reports.plugin.version>
-    <maven.source.plugin.version>3.0.1</maven.source.plugin.version>
+    <maven.source.plugin.version>3.1.0</maven.source.plugin.version>
     <maven.release.plugin.version>2.5.3</maven.release.plugin.version>
     <maven.site.plugin.version>3.7.1</maven.site.plugin.version>
+    <maven.update.checker.version>2.7</maven.update.checker.version>
```

Comment:
To be consistent with the naming of other properties this should be `maven.versions.plugin.version`.

Reply:
done